apps/site/lib/site_web/controllers/places_controller.ex

```diff
@@ -9,15 +9,11 @@ defmodule SiteWeb.PlacesController do
 
   @spec autocomplete(Conn.t(), map) :: Conn.t()
   def autocomplete(conn, %{"input" => input, "hit_limit" => hit_limit_str, "token" => token}) do
```

Comment:
We're no longer using the `token`

Reply:
It seems to work fine without it but I'm wondering if this might cause problems later 😅

Reply:
It seems like the token is always `undefined` from the frontend, not sure if that's a bug but 🤷 I can drop the param for now and add it back if something breaks

Reply:
Maybe it's blank because you don't have a key yet? I don't know... the session token should be generated from the calls to `window.google.maps.places.AutocompleteSessionToken();`; I'm going to guess this will break production once the API starts getting hit with more request than a token-less request would allow, so let's preserve that piece

Reply:
Ah got it, that makes sense. I'll double check this with the google API key set